src/rfc/transfer_encoding_rule.cpp

```diff
@@ -7,7 +7,8 @@
 // Official repository: https://github.com/cppalliance/http_proto
 //
 
-#include <boost/http_proto/rfc/transfer_encoding_rule.hpp>
```

Comment:
I think we should just do away with the whole rfc/ directory and leave it flat

Reply:
Do you want that to be part of this PR or a future one?